public/data/itsarghyadas.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Arghya Das",
+    "bio": "Will keep on updating along with the journey...",
+    "avatar": "https://github.com/itsarghyadas.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/itsarghyadas",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/itsarghyadas",
+        "icon": "twitter"
+      }
+    ]
+  }
```

Comment:
Needs a blank line, as highlighted by GitHub, but this can be updated later

Reply:
okay 👍